### PR TITLE
[IMP] theme_*: adapt theme with new `s_image_hexagonal` snippet

### PR DESCRIPTION
**Pull request by anso-odoo**
*: anelusia, artists, aviato, beauty, bewise, bistro, bookstore, buzzy,
kea, kiddo, monglia, notes, orchid, vehicle, yes

This commit adapts the design of new `s_image_hexagonal` snippet for
multiple themes

task-4094377

requires: https://github.com/odoo/odoo/pull/175813

### CODE REVIEW

File: theme_monglia/views/customizations.xml [hunk 1]
```diff
@@ -148,6 +148,14 @@
     </xpath>
 </template>
 
+<!-- ======== IMAGE HEXAGONAL ======== -->
+<template id="s_image_hexagonal" inherit_id="website.s_image_hexagonal">
+    <!-- Images -->
+    <xpath expr="(//img)[2]" position="attributes">
+        <attribute name="src">/web/image/website.s_text_cover_default_image</attribute>
```
Comment (chgo-odoo) [inline theme_monglia/views/customizations.xml:155]: Not sure about this ? Did you really want to use that image here?

File: theme_notes/views/snippets/s_image_hexagonal.xml
Comment (chgo-odoo) [inline theme_notes/views/snippets/s_image_hexagonal.xml:1]: Readability issue + I'd shorten the title

image

**Review by chgo-odoo — CHANGES_REQUESTED** [review]
Good job Antoine,

Some small comments 👍

File: theme_monglia/views/customizations.xml (hunk 1 above)
Comment (anso-odoo) [reply theme_monglia/views/customizations.xml:155]: I think it's a cache issue on your side. It should be this image

Screenshot 2024-08-21 at 17 29 55

File: theme_kea/views/snippets/s_image_hexagonal.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_image_hexagonal" inherit_id="website.s_image_hexagonal">
+    <!-- Images -->
+    <xpath expr="//img" position="attributes">
+        <attribute name="src">/web_editor/image_shape/website.s_media_list_default_image_1/web_editor/composition/composition_mixed_1.svg</attribute>
```
Comment (bso-odoo) [inline theme_kea/views/snippets/s_image_hexagonal.xml:7]: The `data-shape` attribute should also be adapted.

File: theme_kiddo/views/snippets/s_image_hexagonal.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        A little place of paradise
+    </xpath>
+    <!-- Columns -->
+    <xpath expr="//div[hasclass('o_grid_item_image')]" position="attributes">
+        <attribute name="class" remove="g-col-lg-4 col-lg-4" add="g-col-lg-3 col-lg-3" separator=" "/>
+        <attribute name="style">grid-area: 3 / 2 / 12 / 5; z-index: 1;</attribute>
+    </xpath>
+    <!-- Images -->
+    <xpath expr="//img" position="attributes">
+        <attribute name="src">/web_editor/image_shape/website.library_image_05/web_editor/composition/composition_mixed_1.svg</attribute>
```
Comment (bso-odoo) [inline theme_kiddo/views/snippets/s_image_hexagonal.xml:16]: The `data-shape` attribute should also be adapted.

File: theme_orchid/views/snippets/s_image_hexagonal.xml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="o_cc o_cc4" separator=" "/>
+    </xpath>
+    <!-- Title -->
+    <xpath expr="//h2" position="replace" mode="inner">
+        A place of paradise for flower lovers
+    </xpath>
+    <!-- Images -->
+    <xpath expr="//img" position="attributes">
+        <attribute name="src">/web_editor/image_shape/website.s_masonry_block_default_image_1/web_editor/composition/composition_square_1.svg</attribute>
```
Comment (bso-odoo) [inline theme_orchid/views/snippets/s_image_hexagonal.xml:15]: The `data-shape` attribute should also be adapted.

**Review by bso-odoo — COMMENTED** [review]
Here are a few comments:

File: theme_orchid/views/snippets/s_image_hexagonal.xml (hunk 4 above)
Comment (anso-odoo) [reply theme_orchid/views/snippets/s_image_hexagonal.xml:15]: All three have been fixed, thanks !

**Comment by bso-odoo** [comment]
@robodoo r+

**Pull request closed without merging**